tests/util.test.ts

```diff
@@ -21,7 +21,7 @@ describe('Utility Functions', () => {
   })
 
   afterAll(() => {
-    fs.rmdirSync(tempDir, { recursive: true })
```

Comment:
I was seeing a deprecation warning on this, but looking at the docs, maybe it was just for using `recursive: true` on rmdirSync... https://nodejs.org/api/fs.html#fsrmdirsyncpath-options

I'm not exactly sure what is proper here now...

Reply:
I agree that changing it to `rmSync` looks to be the way to go.  I don't see a deprecation notice on the recursive option for that function, and it does work to delete whole folders recursively.